src/lib/serviceRoutines/deleteIndividualContextEntity.cpp

```diff
@@ -90,7 +93,10 @@ std::string deleteIndividualContextEntity
   }
 
   // 05. Cleanup and return result
+  TIME_STAT_RENDER_START();
```

Comment:
Of course we could do that, we just don't gain anything with it.
It only makes the source code a little more difficult to read.
I don't have a very strong opinion on this. If you really want a new macro for the three calls, sure.
Just let me know.